frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ManagedClusters.tsx

```diff
@@ -415,16 +415,19 @@ export function ClustersTable(props: {
           const unDestroyedClusters = clusters.filter((cluster) => !cluster.isHive)
           setModalProps({
             open: true,
-            alert: (
-              <Alert
-                variant="danger"
-                isInline
-                title={t('You selected {{0}} clusters that cannot be destroyed', [unDestroyedClusters.length])}
-                style={{ marginTop: '20px' }}
-              >
-                <TextContent>{t('They will not be destroyed when you perform this action.')}</TextContent>
-              </Alert>
-            ),
+            alert:
+              unDestroyedClusters.length > 0 ? (
+                <Alert
+                  variant="danger"
+                  isInline
+                  title={t('You selected {{count}} cluster that cannot be destroyed', {
+                    count: unDestroyedClusters.length,
+                  })}
+                  style={{ marginTop: '20px' }}
+                >
+                  <TextContent>{t('They will not be destroyed when you perform this action.')}</TextContent>
```

Comment:
@KevinFCormier What about this message? Is there a way to generate a singular and plural message without using the interpolation `count` variable?

Reply:
Yes, as long as the `count` variable is passed in, it does not actually need to be referenced in the string. See this one for example:
`"This application set uses placement \"{{placement}}\", which is not removable. This placement is shared by the following application set:_plural": "This application set uses placement \"{{placement}}\", which is not removable. This placement is shared by the following application sets:"`

I don't recall if the scripting picks up on this or not, but you can add the `_plural` key manually if not.

Reply:
Ah nice! Thanks @KevinFCormier
@chenz4027 Can you fix it for this message too? Thanks!

Reply:
Sure thanks @fxiang1 @KevinFCormier , I'll update it.

Reply:
Done.